README.md

```diff
@@ -21,3 +21,15 @@ Refer server_example.py
 #### Client
 
 Refer client_example.py
+
+#### Development
+
+Install both production and dev dependencies
+```
+pip install -e ".[dev]"
+```
+
+To re-generate the model classes, run

```

Comment:
Maybe add a line about when they would want to run this. And what needs to be modified to update the model classes. Just a line or two. Nothing too detailed.